test/functional/rpc_fundrawtransaction.py

```diff
@@ -28,6 +28,9 @@ class RawTransactionsTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 4
         self.setup_clean_chain = True
+        # This test isn't testing tx relay. Set whitelist on the peers for
+        # instant tx relay.
+        self.extra_args = [['-whitelist=127.0.01']] * 4
```

Comment:
s/127.0.01/127.0.0.1/ ?

Credit to @practicalswift caught it my PR ;)

Reply:
Maybe also make it more specific: `-whitelist=relay@127.0.0.1`.

You could also add `-whitebind` to cover both inbound and outbound, however for some reason this requires you to specify a port, so you'd have to tweak the test suite for that.

Reply:
Fixed. Thanks!